Common/include/option_structure.hpp

```diff
@@ -1255,94 +1277,98 @@ enum INLET_TYPE {
   DRAG_MINUS_THRUST = 4,  /*!< \brief User specifies the D-T. */
   MASSFLOW = 5,           /*!< \brief User specifies the massflow. */
   POWER = 6               /*!< \brief User specifies the power. */
 };
-static const map<string, ACTDISK_TYPE> ActDisk_Map = CCreateMap<string, ACTDISK_TYPE>
-("VARIABLES_JUMP", VARIABLES_JUMP)
-("BC_THRUST", BC_THRUST)
-("NET_THRUST", NET_THRUST)
-("DRAG_MINUS_THRUST", DRAG_MINUS_THRUST)
-("MASSFLOW", MASSFLOW)
-("POWER", POWER);
+static const MapType<string, ACTDISK_TYPE> ActDisk_Map = {
+  MakePair("VARIABLES_JUMP", VARIABLES_JUMP)
+  MakePair("BC_THRUST", BC_THRUST)
+  MakePair("NET_THRUST", NET_THRUST)
+  MakePair("DRAG_MINUS_THRUST", DRAG_MINUS_THRUST)
+  MakePair("MASSFLOW", MASSFLOW)
+  MakePair("POWER", POWER)
+};
 
 /*!
- * \brief types of geometric entities based on VTK nomenclature
+ * \brief Types of geometric entities based on VTK nomenclature
  */
 enum GEO_TYPE {
-  VERTEX = 1,   	  /*!< \brief VTK nomenclature for defining a vertex element. */
-  LINE = 3,			  /*!< \brief VTK nomenclature for defining a line element. */
-  TRIANGLE = 5, 	  /*!< \brief VTK nomenclature for defining a triangle element. */
+  VERTEX = 1,         /*!< \brief VTK nomenclature for defining a vertex element. */
+  LINE = 3,           /*!< \brief VTK nomenclature for defining a line element. */
+  TRIANGLE = 5,       /*!< \brief VTK nomenclature for defining a triangle element. */
   QUADRILATERAL = 9,  /*!< \brief VTK nomenclature for defining a quadrilateral element. */
   TETRAHEDRON = 10,   /*!< \brief VTK nomenclature for defining a tetrahedron element. */
   HEXAHEDRON = 12,    /*!< \brief VTK nomenclature for defining a hexahedron element. */
-  PRISM = 13,     	  /*!< \brief VTK nomenclature for defining a prism element. */
-  PYRAMID = 14  	  /*!< \brief VTK nomenclature for defining a pyramid element. */
+  PRISM = 13,         /*!< \brief VTK nomenclature for defining a prism element. */
+  PYRAMID = 14        /*!< \brief VTK nomenclature for defining a pyramid element. */
 };
 
 /*!
- * \brief types of objective functions
+ * \brief Types of objective functions
  */
 enum ENUM_OBJECTIVE {
-  DRAG_COEFFICIENT = 1, 	    /*!< \brief Drag objective function definition. */
-  LIFT_COEFFICIENT = 2, 	    /*!< \brief Lift objective function definition. */
+  DRAG_COEFFICIENT = 1,         /*!< \brief Drag objective function definition. */
+  LIFT_COEFFICIENT = 2,         /*!< \brief Lift objective function definition. */
   SIDEFORCE_COEFFICIENT = 3,    /*!< \brief Side force objective function definition. */
-  EFFICIENCY = 4,		        /*!< \brief Efficiency objective function definition. */
+  EFFICIENCY = 4,               /*!< \brief Efficiency objective function definition. */
   INVERSE_DESIGN_PRESSURE = 5,	/*!< \brief Pressure objective function definition (inverse design). */
   INVERSE_DESIGN_HEATFLUX = 6,  /*!< \brief Heat flux objective function definition (inverse design). */
```

Comment:
missed one indentation here :)

Reply:
can't see it for some reason, it shows fine and all editors I have